src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json

```diff
@@ -15,10 +15,16 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00704-04"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.3": {
+      "imports": [
+        "dotnet5.4"
+      ],
+    "dependencies": {
+        "System.Net.Security": "4.0.1-beta-24322-03",
+     },
+   },
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
@StephenBonikowsky @roncain I have to remove this line as this line does package validation for projectN and it fails for System.Net.Security as it does not support projectN. I'm going to validate this change both on ProjectN from projectK and ProjectN using Helix. If they both pass, I think the check here is unnecessary.